.changeset/whatever.md

```diff
@@ -0,0 +1,5 @@
+---
+"typescript-eslint-parser-for-extra-files": minor
```

Comment:
not sure whether a minor version for 0.x needed, @ota-meshi you can make the choice.

```suggestion
"typescript-eslint-parser-for-extra-files": patch
```